dask/bytes/core.py

```diff
@@ -141,10 +141,13 @@ def write_bytes(data, urlpath, name_function=None, compression=None,
         raise NotImplementedError("Unknown protocol for writing %s (%s)" %
                                   (protocol, urlpath))
 
-    files = open_files_write(paths, **storage_options)
-    out = [delayed(write_block_to_file)(v, f, compression, encoding)
-           for (v, f) in zip(data, files)]
-    return out
+    keys = ['write-block-%s' % path for path in paths]
```

Comment:
It should also depend on the data that we're writing to the path, along with any other inputs, like compression or encoding.  If the scheduler sees the same key twice it feels safe disregarding the second one.  In this case we might write some data to paths at one point.  Decide later on to write other data there, and find that the second write didn't take because it had the same key.

Keynames should be short and consistent.  They are used in diagnostics and from time to time in the scheduler.  All of the keys should map to the same value under the `distributed.utils.key_split` function.